shell/renderer/api/atom_api_web_frame.cc

```diff
@@ -110,32 +111,57 @@ class RenderFrameStatus final : public content::RenderFrameObserver {
 
 class ScriptExecutionCallback : public blink::WebScriptExecutionCallback {
  public:
+  // for compatibility with the older version of this, error is after result
+  using CompletionCallback =
+      base::Callback<void(const v8::Local<v8::Value>& result,
```

Comment:
```suggestion
      base::OnceCallback<void(const v8::Local<v8::Value>& result,
```

Reply:
this required using std::move which I'm not confident about, so please check those.